docs/ADR-65-importing-wearables.md

```diff
@@ -0,0 +1,200 @@
+
+The current `asset.json` file has the following format:
+
+```json
+{
+  "id": "4c381333-767a-4036-9959-0fde335a4568",
+  "assetType": "portable-experience",
+  "name": "Portable Experience Example",
+  "description": "My new Portable Experience",
+  "category": "eyewear",
+  "rarity": "mythic",
+  "thumbnail": "glasses.png",
+  "menuBarIcon": "glasses.png",
+  "model": "glasses.glb",
+  "bodyShape": "both"
+}
+```
+
+These properties are used with the purpose described below:
+
+- The `id` property is the ID that the will have or has (if it's being updated) in the builder-server.
+- The `assetType` is used to identify the type of the asset, wether it's a portable experience or a simple wearable for example.
+- The `name` is the name of the wearable, as it's seen in the Marketplace.
+- The `description` is the description of the wearable.
+- The `category` is the category of the wearable (eyewear, hat, etc).
+- The `rarity` is the rarity of the wearable (unique, common, etc).
+- The `thumbnail` is the path to the thumbnail of the wearable.
+- The `menuBarIcon` is the path to the image to use on the “experiences” menu, to represent this portable experience.
+- The `model` is the path to the model of the wearable. This implies that the wearable can have only one model for all representations.
+- The `bodyShape` is the body shape of the wearable. This can only have one value, either `male`, `female` or `both`, implying
+
+The properties `assetType` and `menuBarIcon` are used by the Smart Wearables feature to work, that is, the feature requires an `asset.json` file to be included in the wearables's ZIP file to work. The other properties mentioned before are only used in the import process of wearable in the `Builder UI` and will be used to build wearable.
+
+The current format has some downsides that we need to tackle:
+
+1. It doesn't have the possibility to define representations with different 3D models, that is, a female and a male representation, each one with its 3D model shaped to their body shape.
+2. Wearable tags can't be defined.
+3. It's not possible to define which categories get to be hidden or replaced by the wearable and, as it doesn't allow each representation to be detailed, it doesn't allow overriding hides and replaces for each of representation.
+4. The collection where the wearable is created into can't be specified.
+
+These downsides make the current `asset.json` configuration file not suitable for importing wearables.
+
+## Proposed solution
+
+To solve the downsides of the current format, a new format for the `asset.json`, with the following schema is proposed:
+
+```json
+{
```

Comment:
It is _really_ hard to understand json schemas, can this be changed to typescript types?

Reply:
Changed it!

Reply:
it still looks like a json schema?

Reply:
I'm sorry, I thought that you mean converting the markdown from JSON to TS. My bad. I've just converted it into a TS type.